assets/js/util/is-wp-error.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * WP Error Utilities.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Checks if the provided object is an instance of WP_Error class.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} obj The object to check.
+ * @return {boolean} TRUE if the object has "code", "data" and "message" properties, otherwise FALSE.
+ */
+export function isWPError( obj ) {
+	return typeof obj === 'object' &&
+		obj.hasOwnProperty( 'code' ) && typeof obj.code === 'string' &&
+		obj.hasOwnProperty( 'data' ) && typeof obj.data === 'object' &&
```

Comment:
This may be too strict for `data` which comes from [`get_error_data`](https://developer.wordpress.org/reference/classes/wp_error/get_error_data/) and has a return value of `mixed`

Reply:
Ok, removed type checking for the data property.